services/traefik-forward-auth-mgmt/0.3.9/defaults/cm.yaml

```diff
@@ -34,11 +34,6 @@ data:
       whitelist: []
       enableRBAC: true
       enableImpersonation: true
-      rbacPassThroughPaths:
-        - "/dkp/kubernetes/"
```

Comment:
For kube dashboard tho, I think if we either do this removal here or we remove the Kube dashboard Roles. Otherwise it's a slightly confusing UI.